eng/tools/TypeSpecValidation/dist/TypeSpecValidation.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
```

Comment:
Do we need the js file committed as well?

Reply:
We are going to see if we can run `tsp compile` in a `postinstall` step instead of checking-in the generated JS.  But we think it's OK to check-in the generated JS if it's the only way to make this work.

Reply:
We were able to remove the generated JS, and run compile in a postinstall step in the root package.json.  It did require adding a minimal "shim" JS file but I think this is much better overall.